src/Command/GetGoMRIStatisticsCommand.php

```diff
@@ -98,6 +114,27 @@ protected function execute(InputInterface $input, OutputInterface $output): int
         $io->writeln("Total number of GoMRI datasets submitted since 2021-01-01 until current date " . $totalGomriDatasetSubmittedSince2021Count);
         $io->writeln("Total number of datasets (all data including GoMRI) submitted since 2021-01-01 until current date " . $totalDatasetSubmittedSince2021Count);
 
+        if ($startEpoch != false and $endEpoch != false) { // false on strtotime fail
+            $startDateTime = new \Datetime();
+            // will assume DST-aware central time if not specified.
+            $startDateTime->setTimezone(new \DateTimeZone('America/Chicago'));
+            $startDateTime->setTimestamp($startEpoch);
+            // will include tz in DB string
+            $dbFormatStartTime = $startDateTime->format('Y-m-d H:i:sO');
+
+            $endDateTime = new \Datetime();
+            $endDateTime->setTimezone(new \DateTimeZone('America/Chicago'));
+            $endDateTime->setTimestamp($endEpoch);
+            $dbFormatEndTime = $endDateTime->format('Y-m-d H:i:sO');
```

Comment:
Why not pass DateTime as start end parameters, and let the function format it the way it wants to use it for the query.

You can also create a DateTime in one go from a string.